pkg/controller/operators/olm/operator.go

```diff
@@ -569,13 +575,16 @@ func (a *Operator) syncClusterServiceVersion(obj interface{}) (syncError error)
 			updateErr := errors.New("error updating ClusterServiceVersion status: " + err.Error())
 			if syncError == nil {
 				logger.Info(updateErr)
-				return updateErr
+				syncError = updateErr
+			} else {
+				syncError = fmt.Errorf("error transitioning ClusterServiceVersion: %s and error updating CSV status: %s", syncError, updateErr)
 			}
-			syncError = fmt.Errorf("error transitioning ClusterServiceVersion: %s and error updating CSV status: %s", syncError, updateErr)
 		}
 	}
 
-	a.copyQueueIndexer.Enqueue(outCSV)
+	if !outCSV.IsUncopiable() {
```

Comment:
check for uncopiable before we enqueue, so we don't have to check on the dequeue